src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/client/transformation/model/response/SuccessfulUpdateResponse.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.client.transformation.model.response;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+public class SuccessfulUpdateResponse {
```

Comment:
Are we using this `Successful...Response`? If not, I'd rather just name it sth like `CaseUpdateResponse`.

Reply:
Yes, it is used in UpdateClient. And its name follows the same convention as SuccessfulTransformationResponse.

Reply:
Thanks, that's the convention I was curious about.